website/docs/r/notification_configuration.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_notification_configuration"
+sidebar_current: "docs-resource-tfe-notification-configuration"
+description: |-
+  Manages notifications configurations.
+---
+
+# tfe_notification_configuration
+
+Terraform Cloud can be configured to send notifications for run state transitions. 
```

Comment:
_Very minor, (almost) ignore_ 

Most Terraform resource/data source pages start with something along the lines of:

> The `tfe_notification_configuration` resource can be used to manage [notification configurations](https://www.terraform.io/docs/cloud/api/notification-configurations.html) within Terraform Cloud.

That doesn't look like that's something that's happening in this provider though, so I wouldn't worry about it for now. 😛

Reply:
In this provider those snipped are usually copied from the API doc's as they should already effectively describe what this is for.

Reply:
Yep, that's why I advised to keep.

@lafentres feel free to resolve this at your leisure.